packages/experience-builder-sdk/src/core/designTokenRegistry.ts

```diff
@@ -17,7 +17,9 @@ export const defineDesignTokens = (designTokenDefinition: DesignTokensDefinition
   });
 };
 
-export const getDesignTokenRegistration = (breakpointValue: string) => {
+export const getDesignTokenRegistration = (breakpointValue?: string) => {
+  if (!breakpointValue) return breakpointValue;
+
   let resolvedValue = '';
```

Comment:
Looks like you had some specific edge case in mind. 
But to me the semantics of this looks awkward: 

> take any falsy value and return back _that same_ falsy value 🤷 ...

With such a semantics, it's hard for me to understand the intent of this. (to handle empty strings? to handle undefines? there's also no hint of what is the range of allowed values ....

If it would have been always returning `null` or `false` as result, then I would be able to read it as "when falsy crap comes in, just return false (as marker for no value)" but now if `""`  (empty string) comes, it also returns back the same empty string? 🤷

Reply:
The reason for that is that I don't want to change the behavior. Unfortunately, the new design token logic by @ryunsong-contentful broke some of the existing design variable logic (e.g. the broken rendering of default height and width). This function was added recently as some kind of middleware.
With my change, I want to "turn off" the middleware logic if it's not appliable. So if there is an empty string, undefined, or another falsy value, don't try to transform it anyway. This way, this function is agnostic of any variable-specific default values which might be one of the previously mentioned falsy values.